src/carplay.cs

```diff
@@ -297,13 +417,15 @@ interface CPApplicationDelegate : UIApplicationDelegate {
 	}
 
 	[NoWatch, NoTV, NoMac, iOS (12,0)]
-	[BaseType (typeof (NSObject))]
+	[BaseType (typeof (CPBaseListItem))]
```

Comment:
@spouliot question about API. The class is added in iOS 14, won't this be an issue in previous versions that do have access to the class in iOS 12 and iOS 13?

Reply:
@mandel-macaque iirc, we saw a similar change in a recent PR.

I believe this is ok here because CPBaseListItem is a type of NSObject. Also worth noting - it looks like Apple is going to get rid of CPBaseListItem in favor of CPSelectableListItem...which isn't available and has been filed as a radar here: https://github.com/xamarin/maccore/issues/2298

```	
[Deprecated (PlatformName.iOS, 14, 0)]
[NoWatch, NoTV, NoMac, iOS (14,0)]
[BaseType (typeof (NSObject))]
[DisableDefaultCtor]
interface CPBaseListItem : NSSecureCoding
```

Reply:
@mandel-macaque it's not a problem, nor a breaking change.

We did have a bug (last year or so) where this caused an issue for 32bits code (because the new type would throw an exception as not being available on 32bits OS). This was fixed.